### DISCUSSION TITLE
Show your perspectives !

### DISCUSSION THREAD

**Discussion by FJBDev** [mytourbook/mytourbook#1240] [Show and tell]
What does your MT look like ? I figured we can all learn from each others

Here is the main perspective I use the most. It is mainly designed to analyze and edit my tours (I have other perspectives I use to import tours, compare reference tours, look at statistics):
Note: If you don't want to show your data, don't forget the option below !
```
This will scramble text and values. This is used to create anonymous screenshots

-DscrambleData

```
![image](https://github.com/mytourbook/mytourbook/assets/25235665/49077237-f1cf-4bff-824f-d558fab164b3)

**Comment by wolfgang-ch** [comment]
These are my production main perspectives

![image](https://github.com/mytourbook/mytourbook/assets/1283445/415455f6-4ef1-4f26-8855-9607a12329a7)

![image](https://github.com/mytourbook/mytourbook/assets/1283445/5b25659d-6ce3-435d-b050-65cd78d4a0f9)

**Comment by bern6** [comment]
My perspective

![Main perspective](https://github.com/mytourbook/mytourbook/assets/103836286/4e7642b3-32f1-48f0-9d54-0f5aa89fdcab)

**Comment by aarejogger** [comment]
This is my dashboard, I use MTB just for running.
![Running3](https://github.com/mytourbook/mytourbook/assets/155064080/acb05192-41fb-48bc-8dd5-0cbbb886f01b)

**Comment by Thompson-ongithub** [comment]
What could be the reason -DscrambleData doesn't work for me (Linux)?
To start MTB I type "./mytourbook -DscrambleData" (obviously without the quotation marks) but everything stays unscrambled (OpenJDK 18).

**Reply by wolfgang-ch** [reply]
You have to add the `-vmargs` parameter

![image](https://github.com/mytourbook/mytourbook/assets/1283445/15968982-9e68-4456-ae26-fc332ce7947d)

**Comment by GurVJ** [comment]
This is mine : 
![Capture 001_20240131](https://github.com/mytourbook/mytourbook/assets/64196685/f965812e-85f7-4cd0-83a1-9945f6d53808)
